tests/compiler/dataflow/test_model_assignment.py

```diff
@@ -106,10 +106,13 @@ def test_dataflow_model_attribute_assignment_responsible(dataflow_test_helper: D
 entity Test:
     int n
 end
-implement Test using std::none
+implement Test using none
```

Comment:
this change is repeated a few 100 times

Reply:
I don't get why this results in a performance improvement.

Reply:
loading std is very slow compared to the code under test (much larger, plugins, ...) 

So I make it not load std at all 

(std::Entity is not in std, btw)

Reply:
> (std::Entity is not in std, btw)

Right! I forgot about that.